clang/lib/Sema/SemaAPINotes.cpp

```diff
@@ -583,6 +583,12 @@ static void ProcessAPINotes(Sema &S, FunctionOrMethod AnyFunc,
     }
   }
 
+  if (auto importAs = info.ImportAs) {
+    auto str = "import_" + importAs.getValue();
```

Comment:
Please diagnose invalid attribute values to catch typos in API notes.

Reply:
Actually, if the clang will diagnose automatically after the attribute is added that's fine so you don't need to add a diagnostic here, just make sure clang can actually diagnose a typo here.